docs/Using Fleet/Usage-statistics.md

```diff
@@ -25,6 +25,9 @@ Below is the JSON payload that is sent to Fleet Device Management Inc:
   "hostExpiryEnabled": true,
   "mdmWindowsEnabled": false,
   "liveQueryDisabled": false,
+  "aiFeaturesDisabled": false,
+  "maintenanceWindowsEnabled": true,
```

Comment:
### Dev note
Maintenance windows are considered "enabled" if all of the following is true:
+ configuration has value set for `integrations.google_calendar[0].domain`
+ configuration has value set for `integrations.google_calendar[0].api_key_json` 
+ at least one team has `integrations.google_calendar.enable_calendar_events` set to `true`

Reply:
Good calls on what we use to track this.

Maybe we should we break this up into `maintenanceWindowsConfigured` and `maintenanceWindowEnabled`?

That way we can understand who got as far as hooking up Google Workspace and who got even further to turn maintenance windows on for a team.

Reply:
Good idea 👍